recipes/libv8/build.sh

```diff
@@ -0,0 +1,17 @@
+cd build
+git clone https://chromium.googlesource.com/external/gyp
+cd ..
+sed -i.bak "s/-Werror//g" build/standalone.gypi
+sed -i.bak "s/-Werror//g" build/common.gypi
+python2 build/gyp_v8
+export CPP_INCLUDE_PATH=../src/third_party
+export CPLUS_INCLUDE_PATH=../src/third_party
```

Comment:
export CPLUS_INCLUDE_PATH="${PREFIX}/include:${CPLUS_INCLUDE_PATH}"?

Reply:
It's not set to begin with.